### PR TITLE
Initial version of the reflector

### PR DESCRIPTION
**Pull request by bghgary**
Skeleton version of the reflector.

This change adds the following:
1. Adds a new reflector bridge folder that acts as the server between two reflector clients using WebSockets.
1. Adds a new section in the tools section of the inspector to connect to the reflector.
2. Adds options to the sandbox to run in reflector mode where it will connect to the reflector bridge and wait for the other client to send a scene.

Documentation incoming.

### CODE REVIEW

File: Tools/Gulp/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "license": "(Apache-2.0)",
     "scripts": {
         "install": "cd ../../ && npm install && cd ./Viewer && npm install && cd ../Tools/Gulp/",
-        "build": "gulp --max-old-space-size=8192 --tsLintFix",
+        "build": "gulp --max-old-space-size=8192 --tsLintFix=true",
```
Comment (bghgary) [inline Tools/Gulp/package.json:13]: This change makes it possible to call `npm run build core`.

**Review by RaananW — COMMENTED** [review]
LGTM!

File: sandbox/src/sandbox.tsx [hunk 2]
```diff
@@ -115,15 +133,30 @@ export class Sandbox extends React.Component<ISandboxProps, { isFooterVisible: b
...
 
         return (
             <div id="root">
-                <p id="droptext" ref={this._dropTextRef}>Drag and drop gltf, glb, obj or babylon files to view them</p>
-                <RenderingZone globalState={this._globalState}
-                    assetUrl={this._assetUrl}
-                    autoRotate={this._autoRotate}
-                    cameraPosition={this._cameraPosition}
-                    expanded={!this.state.isFooterVisible} />
+                {renderingZone}
```
[the remark this answers is not in the document]
Comment (deltakosh) [reply sandbox/src/sandbox.tsx:159]: Agreed

**Review by deltakosh — APPROVED** [review]
All good to me except some nit picking with react

File: reflectorBridge/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "babylon-reflector",
+  "version": "0.0.1",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --mode=development",
+    "start": "node ./dist/bundle.js",
+    "debug": "node ./dist/bundle.js"
```
[the remark this answers is not in the document]
Comment (bghgary) [reply reflectorBridge/package.json:8]: Ah, yes, was just playing around. I'll remove one of them.

**Pull request merged**